Dockerfile

```diff
@@ -5,11 +5,11 @@ WORKDIR /cbt
 ENV PYTHONDONTWRITEBYTECODE=1
 ENV PYTHONBUFFERED=1
 
-# create an unprivledged user
+# create an unprivileged user
```

Comment:
Please do not correct my unintentionally intentional spelling mistakes. If I cared about grammar, I'd have become a liberal arts major.

Reply:
My sincerest apologies. I hope you can find it in your heart to forgive me.